### ISSUE TITLE
Training tesseract 4.0 for chinese in windows

### ISSUE THREAD

**Issue by thekingmaker27** [tesseract-ocr/tesseract#1190]
`    lstmtraining --traineddata chi_sim.traineddata --net_spec '[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c111]' --model_output chi --learning_rate 20e-4 --train_listfile chi_sim.training_files.txt --eval_listfile chi_sim.training_files.txt --max_iterations 5000  
`  
I am getting the following error when using the above code

`Warning: given outputs 111 not equal to unicharset of 224. Num outputs,weights in Series:   1,36,0,1:1, 0 Num outputs,weights in Series:   C3,3:9, 0   Ft16:16, 160 Total weights = 160   [C3,3Ft16]:16, 160   Mp3,3:16, 0   Lfys48:48, 12480   Lfx96:96, 55680   Lrx96:96, 74112   Lfx256:256, 361472   Fc224:224, 57568 Total weights = 561472 Built network:[1,36,0,1[C3,3Ft16]Mp3,3Lfys48Lfx96Lrx96Lfx256Fc224] from request [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c111] Training parameters:   Debug interval = 0, weights = 0.1, learning rate = 0.002, momentum=0.5 null char=223 Deserialize header failed: ∩╗┐ Load of page 0 failed! Load of images failed!! Deserialize header failed: E:/chinese.simsun.exp0.lstmF
`

I have specified the lstmf path in chi_sim.training_files.txt Is there anything elese i need to add? Please help me

**Comment by Shreeshrii** [comment]
>Deserialize header failed: ∩╗┐ Load of page 0 failed! Load of images failed!! Deserialize header failed: E:/chinese.simsun.exp0.lstmF

Seems to me the filename extension is incorrect. Should be . lstmf

**Comment by zthdsb** [comment]
@Shreeshrii I encounter the same problem ：
Loaded file jpn.lstm, unpacking...
Warning: LSTMTrainer deserialized an LSTMRecognizer!
Continuing from jpn.lstm
Deserialize header failed: D:/new_6/text.font.exp0.lstmf
Load of page 0 failed!
Load of images failed!!

**Comment by Shreeshrii** [comment]
@zthdsb The problem is because of Windows EOL in the file which has a list of .lstmf files.
